docs/Authentication.md

```diff
@@ -315,3 +315,72 @@ export default withRouter(MyPage);
 {% endraw %}
 
 The `<Authenticated>` component calls the `authClient` function with `AUTH_CHECK` and `authParams`. If the response is a fulfilled promise, the child component is rendered. If the response is a rejected promise, `<Authenticated>` redirects to the login form. Upon successful login, the user is redirected to the initial location (that's why it's necessary to get the location from the router).
+
+
+## Redirect after logout
+
+By default react-admin will redirect to '/login'. There are a couple of ways to change this: 
+
+Add a redirectTo prop to the logoutButton.
+{% raw %} 
```

Comment:
`{% raw %} ` is only necessary if the code sample contains a double curly brace (`{{`), which messes up with Jekyll. It's not the case here, so please remove it.